custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -63,11 +63,13 @@ def person(self):
                 # 'owner_id': self._location.location_id,
                 'update': {
                     'aadhaar_number': self.patient_detail.paadharno,
+                    'age': self.patient_detail.page,
                     'age_entered': self.patient_detail.page,
                     'contact_phone_number': validate_number(self.patient_detail.pmob),
                     'current_address': self.patient_detail.paddress,
                     'current_address_district_choice': self.patient_detail.Dtocode,
                     'current_address_state_choice': self.patient_detail.scode,
+                    'dob_known': False,
```

Comment:
> I think this is set to "no" in the app

fixed

> (would be good to check the other boolean values for what they are set to)

The other booleans are all set by the migration (not the app).  I'll follow up on https://github.com/dimagi/commcare-hq/pull/13293/files/d28dab894c613df238ab727085f226a57d453434#r92884819 in a separate PR

> Also, I think when this is false, we still calculate a DOB in the app, so would be good to do that here too. (for eg check this case.)

Question: since ```dob``` never gets loaded in the app, is it needed?  I suppose it might be used in reports or elsewhere.